src/wazuh_db/schema_upgrade_v11.sql

```diff
@@ -0,0 +1,17 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Febrary 1, 2023.
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+ALTER TABLE sys_hwinfo DROP CHECK sys_hwinfo_chk_1;
+ALTER TABLE sys_hwinfo DROP CHECK sys_hwinfo_chk_2;
+ALTER TABLE sys_hwinfo DROP CHECK sys_hwinfo_chk_3;
+ALTER TABLE sys_hwinfo DROP CHECK sys_hwinfo_chk_4;
+ALTER TABLE sys_hwinfo DROP CHECK sys_hwinfo_chk_5;
```

Comment:
This is not possible to do in SQLite. Also, although this is a small PR, in general forcing a push makes the changes difficult to follow.